static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter/2;
+
     @include mq($until: desktop) {
-        margin-right: $gs-gutter*-1;
-        margin-left: $gs-gutter*-1;
-        padding-right: $gs-gutter;
-        padding-left: $gs-gutter;
-        padding-top: $gs-baseline*4;
+        margin-right: ($content-gutter)*-1;
```

Comment:
In this case I am negating the page margin on the parent element and setting it again as padding on a descendant. In the future this may be refactored so that is no longer the case, in which case the variable can be removed. For the time being though, I think it's valid to use a variable here.

I don't see how the usage of variables apply differently here to how they would in any other code: variables are useful when you need to share a value.

> isn't that the kind of side-effect that we want to remove from our current scss?

I wasn't in the CSS guild meeting. Can you explain?